README.md

```diff
@@ -24,7 +24,8 @@ $ source ~/.cargo/env
 $ rustup install nightly
 $ rustup default nightly
 $ # Install development headers & GCC
-$ sudo apt-get install linux-headers-$(uname -r) gcc
+$ sudo apt-get install linux-headers-$(uname -r) gcc # ubuntu
+$ sudo dnf install kernel-devel-$(uname -r) # fedora
```

Comment:
I also install GCC in that step. Maybe it would be wiser to put this in a separate block like this:
## Build
We need X, Y Z.
On Ubuntu you can install the linux kernel headers like this:
```
sudo apt-get blah blah
```
On fedora like this:
```
sudo dnf blah blah
```
Next install Rust
```
curl | bash
rustup blah nightly
...
```